test/ao/sparsity/test_sparsifier.py

```diff
@@ -45,6 +52,14 @@ def test_constructor(self):
         assert 'test' in sparsifier.module_groups[0]
         assert sparsifier.module_groups[0]['test'] == 3
 
+    def test_step(self):
+        model = Model()
+        sparsifier = ImplementedSparsifier(test=3)
+        sparsifier.enable_mask_update = True
```

Comment:
By default, can enable_mask_update be True, so that in the absence of a scheduler, the mask is updated with every call to step? This way we can avoid requiring users to set this attribute as an additional step.

Reply:
Part of the base sparsifier PR